S3/S3.py

```diff
@@ -1725,6 +1726,10 @@ def send_file(self, request, stream, labels, buffer = '', throttle = 0,
             sha256_hash = checksum_sha256_file(stream, offset, size_total)
         request.body = sha256_hash
 
+        # Provide the checksum with teh request. This is important for buckets that have 
+        # Object Lock enabled.
+        headers['x-amz-checksum-sha256'] = base64.b64encode(sha256_hash.digest()).decode()
```

Comment:
`base64.b64encode(sha256_hash.digest()).decode()`
Can this line be broken down? It is doing a lot all at once. It is hard for readability.
Perhaps we  canpull out the digest operation into a separate sha256_hash_digest variable?
Or maybe this functionality can be separated into a new function in S3/Crypto.py

I know this exact line is recommended online in tutorials but is in my opinion hard to read.
See: https://debugpointer.com/python/sha256-to-base64-python